CHANGELOG.md

```diff
@@ -5,7 +5,24 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
-## [Unreleased]
+## [1.2.0] - 2020-08-05
+### Added
+- <totalcrossLib> param to configuration. By using this parameter, one can deploy a totalcross application that will take place inside the final jar, i.e., KnowCodeXML.jar has a KnowCodeXMLLib.tcz inside it.
+
+- capability to load totalcross java librarys that has its tcz files inside it. Using the same example above, when someone adds a library that has its tcz inside it, the plugin is in charge to take this tcz and place inside the final application.
+
+- parameter <externalResources>. This is a replacement for the file all.pkg. When listing the files inside the this, these files will be added to the root of the final application package. Theses files wont be zipped inside the application tcz files, they accessible inside the final package.
```

Comment:
```suggestion
- parameter <externalResources>. This is a replacement for the file all.pkg. When listing the files inside this, these files will be added to the root of the final application package. Theses files wont be zipped inside the application tcz files, they are accessible inside the final package.
```

Reply:
Didn't understand it well when I read it the first time, I think my suggestion will  make it easier to understand.